.gitignore

```diff
@@ -1,3 +1,5 @@
 __pycache__/
 .env
-env/
\ No newline at end of file
+env/
+**/migrations/*
+!**/migrations/__init__.py
```

Comment:
@APIYOJENNIFER, you do not ignore migration files. Else you will have a DB in a state that you are not sure about. You need to run migrations and commit the resulting files as well

Reply:
@APIYOJENNIFER - hint; in case you want to reduce the number of migration files being created, you can roll back the migrations and create one file. Or look at squashing for advanced use.